sdks/python/manifest.yaml

```diff
@@ -0,0 +1,23 @@
+name: apache_beam
+description: Lyft version of Apache Beam python package
+repository: git@github.com:lyft/beam.git
+credentials: true
+builder:
+  name: s2i
+  params:
+    builder: lyft/s2ipythonlibrary:494bddd033e63c155aababf999650fea120bb664
+tests:
+  - name: unit
```

Comment:
isn't this what triggers the tests

Reply:
See here for an example how the test command is explicit (though it is a Java project): https://github.com/lyft/streamingplatform/blob/master/manifest.yaml#L23